templates/vanilla-sequoia.pkr.hcl

```diff
@@ -62,9 +62,12 @@ source "tart-cli" "tart" {
     # Choose Your Look
     "<wait10s><leftShiftOn><tab><leftShiftOff><spacebar>",
     # Welcome to Mac
-    "<wait10s><tab><spacebar>",
-    # Enable Voice Over
-    "<wait10s><leftAltOn><f5><leftAltOff><wait5s>v",
+    "<wait10s><spacebar>",
```

Comment:
The use of the `tab` key here isn't necessary to get past the welcome screen, so I've removed it.